applications/actian-zen/helm-charts/actianzen/values.yaml

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2020 Actian Corporation
+
+replicaCount: 1
+
+image:
+  name: actianzen
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
```

Comment:
Just a cosmetic comment: There is no need for the empty new lines between the yaml statements

Reply:
Fixed.